### DISCUSSION TITLE
How do we remove all references of a specific tag from all entries in a vault

### DISCUSSION THREAD

**Discussion by doublehelix** [padloc/padloc#615] [Q&A] [answered]
The reason I ask is that when importing from another product, the name of the import file is added to all entries as a tag.
I neither need nor want this to be a tag, as it's so long it covers half the UI when I open the desktop and web plugins.

![image](https://user-images.githubusercontent.com/355130/194964495-2f1fd208-9f6c-4cfa-93e0-7aa833ad7ee0.png)

Also, there's no way to remove an individual tag. Instead it seems I have to manually edit all 1872 entries and remove the tag from each one individually.

I guess my solution would be "Edit" / "Delete" options for tags. (for renaming, typo correction etc. or just plain old get rid of a tag)

**Comment by BrunoBernardino** [comment, marked as answer]
@doublehelix this is a known limitation/feature request I've made at https://github.com/padloc/padloc/issues/351 and that we haven't had a chance to work on, but I'll prioritize it for the next couple of weeks.

**Reply by MaKleSoft** [reply]
Yep, better tag management is on the roadmap already. We should also make sure that

- The length of the tag added during the input is truncated to a maximum length
- Make sure tags with long names are displayed correctly. Tracked here: https://github.com/padloc/padloc/issues/616
